ansible/roles/kong-api/defaults/main.yml

```diff
@@ -16053,4 +16071,233 @@ kong_apis:
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+      - surveyAccess
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ x_large_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: signupsurveySolutions
+    uris: "{{ surveys_service_prefix }}/template/api/v1/signup"
+    upstream_url: "{{ ml_survey_service_url }}/template/api/v1/signup"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+      - surveyAccess
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ x_large_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: errorDownloadofsurveySolutions
+    uris: "{{ surveys_service_prefix }}/template/api/v1/errDownload"
+    upstream_url: "{{ ml_survey_service_url }}/template/api/v1/errDownload"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+      - surveyAccess
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ x_large_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: userRolesList
+    uris: "{{ surveys_service_prefix }}/template/api/v1/userRoles/list"
+    upstream_url: "{{ ml_survey_service_url }}/template/api/v1/userRoles/list"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+      - surveyAccess
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ x_large_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: surveyaddSampletemplate
+    uris: "{{ surveys_service_prefix }}template/api/v1/add/sampleTemplate"
+    upstream_url: "{{ ml_survey_service_url }}template/api/v1/add/sampleTemplate"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+      - surveyAccess
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ x_large_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: surveyupdateSampleTemplate
+    uris: "{{ surveys_service_prefix }}/template/api/v1/update/sampleTemplate/{template_id}"
```

Comment:
@anudeep1706  {template_id} is not require i think, is this kind of endpoint added in this or not